src/components/Carousel.tsx

```diff
@@ -67,18 +64,14 @@ export function Carousel({ items }: { items: string[] }) {
             key={i}
             href={`/work/item-${i}`}
             onClick={() => dispatch({ type: "projectIndex", value: i })}
-            initial={{ opacity: state.projectIndex === i ? 1 : 0 }}
-            exit={{ opacity: state.projectIndex === i ? 1 : 0 }}
-            transition={transition}
-            animate={{ opacity: 1 }}
           >
-            <motion.img
+            <MotionImage
```

Comment:
rather than an image, it should just be a wrapper that works for image or video

Reply:
That's coming in the video branch